packages/flutter_localizations/lib/src/l10n/widgets_tr.arb

```diff
@@ -4,5 +4,12 @@
   "reorderItemUp": "Yukarı taşı",
   "reorderItemDown": "Aşağı taşı",
   "reorderItemLeft": "Sola taşı",
-  "reorderItemRight": "Sağa taşı"
+  "reorderItemRight": "Sağa taşı",
+  "copyButtonLabel": "Kopyala",
+  "cutButtonLabel": "Kes",
+  "lookUpButtonLabel": "Ara",
+  "searchWebButtonLabel": "Web'de Ara",
+  "shareButtonLabel": "Paylaş",
+  "pasteButtonLabel": "Yapıştır",
+  "selectAllButtonLabel": "Tümünü seç"
```

Comment:
how come we didn't have these localizations before for flutter context menu?

Reply:
We had them for Material and Cupertino, but now we want them for Widgets (the parent of Material and Cupertino), because SystemContextMenu is in the Widgets library.